Summarize the Visual Content Considerations (#4561)
ppy/osu-wiki: wiki/Rules/Visual_Content_Considerations/en.md

When in doubt, aim to accommodate the collective values shared by the largest portion of the osu! community in general.

## Criteria

- **Images with any sexual content involving (or are targeted at) minors are not allowed.**
- **Images with excessive violence are not allowed.**
- **Images with significant sexual innuendo are not allowed.**
- **Images with explicit content or sexual posturing are not allowed.**
- **Images with any erotic content or graphic displays of sexuality are not allowed.**
- **Images with any examples of drug use or preparation are not allowed.**
- **Images with depictions of any deliberately inflammatory political, cultural, religious or social content are not allowed.**

## Exceptions

- Images with mild to moderate depictions of violence may be allowed.
- Images with depictions of artistic nudity in the context of classical artworks or other tasteful depictions of the human body may be allowed.
- Images with depictions of mild romance, flirting or intimate affection may be allowed.

## Getting assessed

> To check to see if an image is appropriate, **please submit a formal request to any Beatmap Nominator (for beatmap related content) or member of the Global Moderation Team (for anything).**

Reviewer: ```suggestion
To check to see if an image is appropriate, **please submit a formal request to any [Beatmap Nominator](/wiki/People/The_Team/Beatmap_Nominators) (for beatmap related content) or member of the [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team) (for anything).**
```

The issue will then be raised in a dedicated content reviewing channel for discussion and a voting period.

The voting period will not last longer than 7 days.

The vote must pass with at least a 60% majority. Votes that do not gain this majority will be considered too contentious to safely allow, and will be automatically denied.

> If a given image does not pass a vote, that decision is **final** and may not be overridden or raised for discussion again, unless the image is altered to attempt to resolve the problems with it.

Reviewer: you've been talking about an example vote scenario, so I think it'd be better to talk about "the" situation rather than introducing other hypothetical ones.

(i can't write this as a suggestion cuz github)
```markdown
The image will be allowed if the vote passes with at least a 60% majority. Otherwise, it will be considered too contentious to safely allow, and automatically denied. The decision by vote is **final** and may not be overridden or raised for discussion again, unless the image is altered in attempt to resolve the problems with it.
```

> The game's administration (the osu!team) reserves the right to unilaterally declare a piece of content either valid or invalid at their discretion, regardless of any previous voting held on the matter.

Reviewer: ```suggestion
The [osu!team](/wiki/People/The_Team) reserves the right to unilaterally declare a piece of content either valid or invalid at their discretion, regardless of any previous voting held on the matter.
```

## Criteria

Images or visual elements containing any of the following are **not allowed**:

- **sexual content involving or targeted at minors**
- **excessive violence, gore, dismemberment, decapitation, or maiming**
- **significant sexual innuendo**
- **sexual posturing or solicitation**
- **erotic content or graphic displays of sexuality**
- **examples of drug use, preparation, or identification with illicit drug subculture**
- **depictions of deliberately inflammatory political, cultural, religious, or social content**

## Exceptions

Images or visual elements containing examples of the following may be considered acceptable:

- mild to moderate depictions of violence (small amounts of blood, non-critical or superficial wounding)
- artistic nudity (in the context of classical artwork, or work related to the element, such as album art with minor nudity present)
- mild romance, flirting or intimate affection (hugging, embracing, kissing without sexual connotations, hand-holding, etc.)
- swimsuits of reasonable design (standard swimwear with general coverage, ordinary bikini)

## Getting your image assessed

> If you are unsure about an image that is going to be included in a beatmap in any way, **please submit a formal request to any [Beatmap Nominator](/wiki/People/The_Team/Beatmap_Nominators), member of the [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team) or member of the or [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team).** Images that are used elsewhere (avatars, profile covers, userpages, etc) are **not** subject to the vote listed above and may be removed at any time at the discretion of the Global Moderation Team.

Reviewer 2: ```suggestion
If you are unsure about an image that is going to be included in a beatmap in any way, **please submit a formal request to any [Beatmap Nominator](/wiki/People/The_Team/Beatmap_Nominators), member of the [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team) or member of the [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team).** Images that are used elsewhere (avatars, profile covers, userpages, etc) are **not** subject to the voting procedure explained below and may be removed at any time at the discretion of the Global Moderation Team.
```
as there isn't any mention regarding the voting procedure above

# Visual content considerations

As per [Community Rule 5](/wiki/Rules#community-rules), all visual content submitted or uploaded to osu! must be considered as appropriate for an all-ages game.

This includes:

- submitted beatmaps
- profile details (avatars, cover images)
- any other attachments or hyperlinks (forum posts, comments, beatmap discussion posts)

In the interest of keeping things as clean as possible, Community Rule 5 applies to anything that could be reasonably considered "questionable".

## Important things to consider

> All pictures should be judged as they are, without factoring in any other cases before them. Decide based on what you see.

Reviewer 3: any reason these are separate lines and not one paragraph?

Author: to aid primarily in comprehension. having points split up like this in other documents appears to aid significantly in uptake and understanding (from experience) like bullet points, but without the bullets 

it's a stylistic thing though, can change if you feel it's pointless

Reviewer 3: I've never seen this done before, is this a common thing? I didn't even realise markdown would handle this scenario and add paragraphs for each line (as it at least seems to be doing in the github preview).

Author: simple english wikipedia is full of examples if you need them

unsure re: markdown, though i can't see why it wouldn't handle this properly?

Reviewer 3: I guess it's fine.

> All pictures should be judged as they are, without factoring in any other cases before them. Decide based on what you see.

Reviewer 4: this also gives green light to media sourced from R-18 works that appears SFW by itself when cropped, is that correct?

Reviewer: I think that makes sense considering nobody seems to have had issue with using sfw character art from eroge, which is in like... thousands of ranked maps

Author: "all media should be judged in a vacuum" applies here and any references back to NSFW content can be addressed in source or by application of a forthcoming NSFW filter

Please also understand that we are a global, multicultural community with widely varying thoughts, opinions, and values on the topic of what is and isn't allowed.

When in doubt, aim to accommodate the collective values shared by the largest portion of the osu! community in general.

## Criteria

Images or visual elements containing any of the following are **not allowed**:

> - **any sexual content involving (or is targeted at) minors**

Reviewer: ```suggestion
- **sexual content involving or targeted at minors**
```

> - **excessive violence, gore, dismemberment, decapitation or maiming**

Reviewer: ```suggestion
- **excessive violence, gore, dismemberment, decapitation, or maiming**
```

- **significant sexual innuendo**
- **sexual posturing or solicitation**
- **erotic content or graphic displays of sexuality**

> - **examples of drug use, preparation, or identification with any illicit drug subculture**

Reviewer: ```suggestion
- **examples of drug use, preparation, or identification with illicit drug subculture**
```

> - **depictions of deliberately inflammatory political, cultural, religious, or social content**

Reviewer 4: I think it's excessively bold, may go without it, considering the sentence above

Author: gonna leave it bold since it needs to draw people's attention

> - **depictions of any deliberately inflammatory political, cultural, religious or social content**

Reviewer: ```suggestion
- **depictions of deliberately inflammatory political, cultural, religious, or social content**
```

## Exceptions

Images or visual elements containing examples of the following may be considered acceptable:

- mild to moderate depictions of violence (small amounts of blood, non-critical or superficial wounding)

> - artistic nudity (in the context of classical artwork, or work related to the element, such as album art with minor nudity prese`n`t)

Reviewer 2: ```suggestion
- artistic nudity (in the context of classical artwork, or work related to the element, such as album art with minor nudity present)
```
fix typo

> - mild romance, flirting or intimate affection (hugging, embracing, kissing without sexual connotations, hand-holding, etc)

Reviewer: ```suggestion
- mild romance, flirting or intimate affection (hugging, embracing, kissing without sexual connotations, hand-holding, etc.)
```

- swimsuits of reasonable design (standard swimwear with general coverage, ordinary bikini)

## Getting your image assessed

> If you are unsure about an image that is going to be included in a beatmap in any way, **please submit a formal request to any [Beatmap Nominator](/wiki/People/The_Team/Beatmap_Nominators) or member of the [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team).**

Reviewer 4: ```suggestion
If you are unsure about an image that is going to be included in a beatmap in any way, **please submit a formal request to any [Beatmap Nominator](/wiki/People/The_Team/Beatmap_Nominators), or member of the [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team) or [Nomination Assessment Team](/wiki/People/The_Team/Global_Moderation_Team).**
```

> If you are unsure about an image that is going to be included in a beatmap in any way, **please submit a formal request to any [Beatmap Nominator](/wiki/People/The_Team/Beatmap_Nominators), member of the [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team) or member of the [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team).**

Reviewer 3: can this be replaced with at very least a google form or something, which posts to discord? seems pretty annoying to have people reaching out in private to moderators.

Author: talking with pishi now about getting it added to existing BN site report functionality which is much less prone to abuse

> The image will be raised to a vote that must pass by a considerable majority, or will be automatically denied. The result of any vote is **final** and may not be overridden or raised for discussion again, unless the image is significantly altered in an attempt to resolve the problems with it.

Reviewer 4: while I understand the first sentence, it reads like it's missing "it" in the last part

Reviewer: I was thinking about this too, unfortunately it's still ambiguous whether "it" would mean the image or the image's vote. and if you interpret that as "the image", the sentence reads like: "it will be raised to a vote [...] OR will be automatically denied"

I think this could be better as two sentences, one to say there's a vote, another to say what happens when it passes/fails

Author: altered the wording here

> Any images used elsewhere on osu! (avatars, profile covers, userpage images, etc) are not subject to the voting process outlined above, and are instead left to the discretion of the Global Moderation Team.

Reviewer: ```suggestion
Images used elsewhere on osu! (avatars, profile covers, userpage images, etc) are not subject to the voting process outlined above, and are instead left to the discretion of the Global Moderation Team.
```

> Any images used elsewhere on osu! (avatars, profile covers, userpage images, etc) are not subject to the voting process outlined above, and are instead left to the discretion of the Global Moderation Team.

Reviewer: kinda feels like this should come first in the section, since it's more general and seems expected that GMT handle this stuff. then you can go on to mention how beatmaps are a special case with the voting and BN involvement yada yada
